build.sh

```diff
@@ -7,25 +7,25 @@ cd build
 
 cmake \
   -G Ninja \
-  -DUSE_SYSTEM_EIGEN=ON \
-  -DENABLE_OPENGL=OFF \
-  -DENABLE_MANTIDPLOT=FALSE \
-  -DENABLE_WORKBENCH=FALSE \
-  -DENABLE_OPENCASCADE=FALSE \
-  -DPYTHON_EXECUTABLE="$CONDA_PREFIX/bin/python" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DUSE_JEMALLOC=OFF \
```

Comment:
Why was this done?

Reply:
Only used via `mantidpython` wrapper scripts on linux - For reference https://github.com/mantidproject/conda-recipes/pull/55#discussion_r545168667